library/src/tensor_operation_instance/gpu/CMakeLists.txt

```diff
@@ -322,7 +330,7 @@ FOREACH(subdir_path ${dir_list})
 
         if(MIOPEN_REQ_LIBS_ONLY)
             message(STATUS "Removing all sources that are not required for MIOpen")
-            if(NOT "${cmake_instance}" MATCHES "conv")
+            if(NOT "${cmake_instance}" MATCHES "device_conv|device_grouped_convnd|device_grouped_conv2d_nhwgc|device_grouped_conv3d_ndhwgc")
```

Comment:
Random place: Can we exclude int8 data type and mixed precision (fp8, bf8, bf16_f32_bf16 for wrw) for miopen build?